frontend/src/pages/modelRegistry/screens/RegisterModel/utils.ts

```diff
@@ -100,6 +100,7 @@ export const registerVersion = async (
       author,
       modelFormatName: formData.sourceModelFormat,
       modelFormatVersion: formData.sourceModelFormatVersion,
+      modelLocationType: formData.modelLocationType,
```

Comment:
Hey... so I came by here to leave that last comment (I scrolled past the TODO comments again when looking at this file to answer someone's question) and I noticed this line above --- `modelLocationType` isn't one of the properties of a ModelArtifact in [the swagger docs](https://editor.swagger.io/?url=https://raw.githubusercontent.com/opendatahub-io/model-registry/main/api/openapi/model-registry.yaml). I don't think adding this does anything.

Reply:
Removed this.. yes this is not being used anywhere. I maybe added this for testing and forgot to remove. Thanks for catching this!